pkg/dns/ibm/private/dnssvcs_provider.go

```diff
@@ -190,9 +194,15 @@ func (p *Provider) createOrUpdateDNSRecord(record *iov1.DNSRecord, zone configv1
 		log.Info("Warning: TTL must be one of [1 60 120 300 600 900 1800 3600 7200 18000 43200]. RecordTTL set to default", "default DSNSVCS record TTL", defaultDNSSVCSRecordTTL)
 		record.Spec.RecordTTL = defaultDNSSVCSRecordTTL
 	}
+	// We only support one target, warn the user.
+	if len(record.Spec.Targets) > 1 {
```

Comment:
Why not add this to [ValidateInputDNSData](https://github.com/openshift/cluster-ingress-operator/blob/master/pkg/dns/ibm/common.go#L88-L90)?  Shouldn't this error stop the addition of the record, rather than just logging it?

Reply:
We could add a Boolean argument to `ValidateInputDNSData` for allowing/rejecting multiple targets.  That said, `ValidateInputDNSData` always felt weird to me anyway; it seems like the validation should be moved into the controller, or even enforced by API validation.  The approach in this PR seems fine to me.  Eventually, it would be nice to follow up with some rethinking and refactoring of the validation logic.